### ISSUE TITLE
False positive for E124 and E128

### ISSUE THREAD

**Issue by myint** [PyCQA/pycodestyle#100]
``` python
d = {  # comment
    1: 2
}
```

```
$ pep8 a.py
a.py:2:5: E128 continuation line under-indented for visual indent
a.py:3:1: E124 closing bracket does not match visual indentation
```

Removing the comment makes the problem go away.

``` python
d = {
    1: 2
}
```

```
$ pep8 a.py
```
